### PR TITLE
Changelog updates for the 1.0.0 release.

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
 
 ## Unreleased:
 
+## Version 1.0.0 - 2021-02-26
+
+### General
```
Comment (bogdandrutu) [inline CHANGELOG.md:7]: Would be good to maybe have a link to the stable packages?

Comment (jkwatson) [reply CHANGELOG.md:7]: good idea, yes... can't really add that until we do a publish and we can point at a published bom/pom, though, i think.

Comment (bogdandrutu) [reply CHANGELOG.md:7]: Ok, probably we can do it in the release notes. Or we can have an entry to the stable bom in the readme :) Can be done later.

**Review by bogdandrutu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jkwatson) [reply CHANGELOG.md:7]: I added it here.

**Pull request merged**